data/battle_frontier_1.s

```diff
@@ -20,10 +41,30 @@ gUnknown_0860CE84:: @ 860CE84
 	.incbin "baserom.gba", 0x60ce84, 0x10
 
 gUnknown_0860CE94:: @ 860CE94
-	.incbin "baserom.gba", 0x60ce94, 0x20
+	window_template 0x00, 0x00, 0x03, 0x08, 0x10, 0x0f, 0x0010
+	window_template 0x00, 0x16, 0x03, 0x08, 0x10, 0x0f, 0x0090
+	window_template 0x00, 0x08, 0x01, 0x0e, 0x02, 0x0f, 0x0110
+	window_template 0xFF, 0x00, 0x00, 0x00, 0x00, 0x00, 0x0000
```

Comment:
0xFF, 0x00, 0x00, 0x00, 0x00, 0x00, 0x0000 should be replaced with window_template_terminator.